jekyll/_cci2/build-artifacts.adoc

```diff
@@ -6,6 +6,7 @@ version:
 = Build Artifacts
 :page-layout: classic-docs
 :page-liquid:
+:page-description: Build artifacts persist data after a job is completed and can be used for longer-term storage of your build process outputs.
```

Comment:
Sorry must have been crossed wires I meant use the Jekyll front matter rather than asciidoc attricbutes for these descriptions. Let's leave them like this though for now and I will test the PDFs aren't negatively affected. Should be fine I think :-D